src/data-extractor/abstractDataExtractor.ts

```diff
@@ -12,7 +12,7 @@ export abstract class AbstractDataExtractor {
 
   abstract createPrimitiveVariable(variable: DebugProtocol.Variable): PanelViewVariable;
   abstract createPrimitiveValue(variable: DebugProtocol.Variable): PrimitiveValue;
-  abstract createVariableId(variable: DebugProtocol.Variable): string;
+  abstract createVariableId(variable: DebugProtocol.Variable, parentId?: string | undefined): string;
```

Comment:
```suggestion
  abstract createVariableId(variable: DebugProtocol.Variable, parentId?: string): string;
```
The `?` implies that it can be undefined.